invoices/utils.py

```diff
@@ -40,3 +43,11 @@ def __call__(self, value):
 
     def __eq__(self, other):
         return self.schema == other.schema
+
+
+def zip_files(files):
+    outfile = BytesIO()
+    with zipfile.ZipFile(outfile, 'w') as zf:
+        for file in files:
+            zf.writestr(f"{get_valid_filename(file[0])}.xml", file[1])
```

Comment:
From what I understood the name of the xml (and of the zip file) needs to follow a specific format, there are some info here: https://www.fatturapa.gov.it/export/fatturazione/it/c-11.htm (in italian)

Reply:
Done! 👍